care/facility/migrations/0025_auto_20200326_0320.py

```diff
@@ -0,0 +1,99 @@
+# Generated by Django 2.2.11 on 2020-03-26 03:20
+
+import datetime
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('facility', '0024_auto_20200325_0311'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='patientregistration',
+            name='created_date',
+            field=models.DateTimeField(default=datetime.datetime(2020, 3, 26, 3, 20, 32, 409615)),
+        ),
+        migrations.AddField(
+            model_name='patientregistration',
+            name='modified_date',
+            field=models.DateTimeField(auto_now=True),
+        ),
+        migrations.AddField(
+            model_name='patientteleconsultation',
+            name='deleted',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='patientteleconsultation',
+            name='modified_date',
+            field=models.DateTimeField(auto_now=True),
+        ),
+        migrations.AlterField(
```

Comment:
Do we actually need these AlterField? I understand that it is auto-generated.
I am just curious, if it might overwrite the earlier created_date and modified_date?
Can you confirm that this won't happen.